Dockerfile

```diff
@@ -7,22 +7,29 @@ ENV GOBIN=/bin
 COPY --from=bufbuild/buf:1.34 /usr/local/bin/buf /usr/local/bin/buf
 COPY --from=arigaio/atlas:0.25.0-distroless /atlas /usr/local/bin/atlas
 
-RUN apk update && \
+RUN apk update; \
```

Comment:
why not here doc would make it much easier to read

Reply:
I agree, changed